packages/marketprice/src/alarms/mod.rs

```diff
@@ -21,23 +25,17 @@ pub mod errors;
 
 pub type AlarmsCount = u32;
 
-pub struct PriceAlarms<'m> {
-    alarms_below_namespace: &'m str,
-    alarms_above_namespace: &'m str,
-    index_below_namespace: &'m str,
-    index_above_namespace: &'m str,
-}
-
 #[derive(Clone, Serialize, Deserialize, Debug)]
 struct AlarmStore(CoinDTO<SwapGroup>);
```

Comment:
let's take this opportunity to change the name to a more meaningful one, for example, `NormalizedPrice`